### PR TITLE
Add performance testing framework

### PR DESCRIPTION
**Pull request by pmacik**
Signed-off-by: Pavel Macík <pavel.macik@gmail.com>

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#docs) 
  included if any changes are user facing
- [ ] [Tests](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#tests)
  included if any functionality added or changed. For bugfixes please include tests that can catch regressions
- [ ] All acceptance test scenarios included in the PR which verifies a bugfix or a requested feature reported by a non-member are tagged with `@external-feedback` tag.
- [ ] Follows the [commit message standard](https://github.com/redhat-developer/service-binding-operator/blob/master/CONTRIBUTING.md#commits)

### CODE REVIEW

File: test/performance/kpidiff.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                                c = 0.0
+                        else:
+                            c = b / a - 1.0
+                        if raw:
+                            value = c
+                        else:
+                            if c > 0:
+                                sign = "+"
+                            else:
+                                sign = ""
```
Comment (sadlerap) [inline test/performance/kpidiff.py:49]: If `a` is larger than `b` and `a != 0`, then `c` will be negative.  Shouldn't that be reflected in the sign?
```suggestion
                                sign = "-"
```

Comment (pmacik) [reply test/performance/kpidiff.py:49]: > If `a` is larger than `b` and `a != 0`, then `c` will be negative.  Shouldn't that be reflected in the sign?
> 
> ```suggestion
> 
>                                 sign = "-"
> 
> ```

The negative sign is shown normally for negative values, this is mainly for showing the `+` sign at positive values.

**Review by sadlerap — APPROVED** [review]
/lgtm

**Pull request merged**